pkg/beacon/relay/groupselection/groupselection.go

```diff
@@ -56,223 +46,214 @@ func CandidateToNewGroup(
 	if err != nil {
 		return err
 	}
-	initialSubmissionTickets, reactiveSubmissionTickets, err :=
-		generateTickets(
-			newEntry.Bytes(),
-			staker.Address(),
-			availableStake,
-			chainConfig.MinimumStake,
-			naturalThreshold(chainConfig),
-		)
+
+	tickets, err := generateTickets(
+		newEntry.Bytes(),
+		staker.Address(),
+		availableStake,
+		chainConfig.MinimumStake,
+	)
 	if err != nil {
 		return err
 	}
 
-	logger.Infof(
-		"generated [%v] tickets for initial submission phase and [%v] "+
-			"tickets for reactive submission phase",
-		len(initialSubmissionTickets),
-		len(reactiveSubmissionTickets),
-	)
+	logger.Infof("generated [%v] tickets", len(tickets))
 
 	return startTicketSubmission(
-		initialSubmissionTickets,
-		reactiveSubmissionTickets,
+		tickets,
 		relayChain,
 		blockCounter,
 		chainConfig,
 		startBlockHeight,
 		onGroupSelected,
+		ticketSubmissionRoundDuration,
```

Comment:
This is a constant, we don't need to pass it as a parameter.

Reply:
For test purposes, we need a smaller number in order to avoid long test execution.

Reply:
Ok, this is no longer an argument as we changed the round to `3` blocks. I will change.